lib/va1010_forms/utils.rb

```diff
@@ -38,5 +37,24 @@ def soap
     def override_parsed_form(parsed_form)
       HCA::OverridesParser.new(parsed_form).override
     end
+
+    private
+
+    def submission_body(formatted_form)
+      content = Gyoku.xml(formatted_form)
+      submission_body = soap.build_request(:save_submit_form, message: content).body
+      log_payload_info(formatted_form, submission_body)
+
+      submission_body
+    end
+
+    def log_payload_info(formatted_form, submission_body)
+      form_name = formatted_form['va:form']['va:formIdentifier']['va:value']
+      attachment_count = formatted_form['va:form']['va:attachments']&.length || 0
+
+      Rails.logger.info("Payload for submitted #{form_name}: " \
```

Comment:
Add info log here to better allow for debugging these errors in the future. I also did a small refactor to pull out some of the complexity in the `es_submit` method that does not have any functional changes.